chrome/skin/style.css

```diff
@@ -85,3 +85,13 @@ textbox#urlbar {
 #universal-search-recommendation-label {
   color: #BBB;
 }
+
+/* highlight color, applied when the recommendation is highlighted */
+
+#universal-search-recommendation.highlight {
+  background-color: #3b8ce3;
```

Comment:
Can we use the inbuilt CSS here instead for a little more native-feeling UI? It looks like the appropriate highlight styles are applied via `.autocomplete-richlistitem[selected="true"]`:

https://dxr.mozilla.org/mozilla-central/source/toolkit/themes/linux/global/autocomplete.css#110
https://dxr.mozilla.org/mozilla-central/source/toolkit/themes/osx/global/autocomplete.css#96
https://dxr.mozilla.org/mozilla-central/source/toolkit/themes/windows/global/autocomplete.css#104
https://dxr.mozilla.org/mozilla-central/source/toolkit/themes/windows/global/autocomplete.css#112